python/kserve/kserve/model.py

```diff
@@ -296,16 +274,7 @@ async def postprocess(
 
     async def _http_predict(
         self, payload: Union[Dict, InferRequest], headers: Dict[str, str] = None
-    ) -> Dict:
-        protocol = "https" if self.use_ssl else "http"
-        predict_url = PREDICTOR_URL_FORMAT.format(
-            protocol, self.predictor_host, self.name
-        )
-        if self.protocol == PredictorProtocol.REST_V2.value:
-            predict_url = PREDICTOR_V2_URL_FORMAT.format(
-                protocol, self.predictor_host, self.name
-            )
-
+    ) -> Union[Dict, InferResponse]:
```

Comment:
Before it was only returning dict, this could be a breaking change if the transformer is depending on the dict type

Reply:
I don't think it will be a breaking change. This is a internal method used by predict function. Previously, Predict function gets the dict and converts it back to InferResponse, now the _http_predict directly returns the InferResponse